.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
Wrap mode will increase your DEV bundle size. If you only ship the development build (which I don't recommend), then this PR is probably not doing any good.